core/service/config/default_1.toml

```diff
@@ -56,6 +56,14 @@ path = "./keys"
 # "symm" and "asymm")
 # root_key_spec = "symm"
 
+# This is the storage backend up the backup vault.
+# If configured, most operations that writes to the private vault
+# will also be written to the backup vault.
+# We do not write PrssState to the backup vault since it can be regenerated easily
+# [backup_vault]
```

Comment:
Why is it outcommented here?

Reply:
I think what's in `default_1.toml` are the default vaules. Not entirely sure if it should be configured by default?